be/src/olap/rowset/alpha_rowset.cpp

```diff
@@ -219,6 +219,20 @@ OLAPStatus AlphaRowset::make_snapshot(const std::string& snapshot_path,
     return OLAP_SUCCESS;
 }
                                     
+OLAPStatus AlphaRowset::copy_files_to_path(const std::string& dest_path,
+                                           std::vector<std::string>* success_files) {
```

Comment:
should add copy_files_to_path to parent class

Reply:
Already added.